### DISCUSSION TITLE
Final prediction Unit

### DISCUSSION THREAD

**Discussion by jiny0131** [PaddlePaddle/PaddleSpatial#22] [BaiduKDDCup2022] [answered]
In the baseline code, the final prediction use the unit "Mega watt" instead of "Kilo watt".
However in the evaluation part of the dataset report does not mention this point.
Which unit should we use to calculate the loss?

**Comment by zhoujingbo** [comment, marked as answer]
Hi, 

We will use the Mega watt, and this is declared in our baseline code ：

> https://github.com/PaddlePaddle/PaddleSpatial/blob/main/apps/wpf_baseline_gru/evaluation.py  line 83

In any way, thanks a lot for pointing out this point, we will update our report to make this clear.

Thanks a lot.
Regards
Jingbo
